app/models/case_log.rb

```diff
@@ -268,6 +268,14 @@ def benefits_unknown?
     hb == 3
   end
 
+  def this_landlord?
```

Comment:
What does `this` refer to here?

Reply:
Yeah not a blocker for this PR at all, just a little vague how that's worded in the data dictionary so would be good to make it clearer here.